anomalib/config/config.py

```diff
@@ -175,6 +150,50 @@ def get_configurable_parameters(
             Custom changes to these modules will be needed to prevent errors and/or unpredictable behaviour."
         )
 
+    if config.dataset.format == "folder" and "split_ratio" in config.dataset.keys():
+        warn(
+            "The 'split_ratio' parameter is deprecated and will be removed in v0.4.0. Please use "
+            "'normal_split_ratio' instead."
+        )
```

Comment:
It would be easier for us to track what to deprecate if we use DeprecationWarning
```suggestion
        warn(
            DeprecationWarning(
                "The 'split_ratio' parameter is deprecated and will be removed in v0.4.0. Please use "
                "'normal_split_ratio' instead."
            )
        )
```